test_driver/transitions_perf_test.dart

```diff
@@ -210,11 +222,31 @@ void main([List<String> args = const <String>[]]) {
       }
     });
 
+    test('only Crane', () async {
+      if (!isTestingCraneOnly) return;
+
+      // Collect timeline data for just the Crane study.
+      final timeline = await driver.traceAction(
+        () async {
+          await runDemos(['crane@study'], driver);
+        },
+        streams: const <TimelineStream>[
+          TimelineStream.dart,
+          TimelineStream.embedder,
+        ],
+      );
+
+      final summary = TimelineSummary.summarize(timeline);
+      await summary.writeSummaryToFile('transitions-crane', pretty: true);
+    });
```

Comment:
The `all demos` test has a timeout, should this one?

Reply:
I don't see why not, I'll add one